jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -170,7 +173,7 @@ private static void enter(Continuation cont) {
 		try {
 			cont.runnable.run();
 		} finally {
-			cont.finished = true;
+			cont.setFinished();
 			yieldImpl();
```

Comment:
is there a reason why `yieldImpl` cant just take a parameter? (ie true == finished) this way we can avoid the extra JNI call